backend/satellite_tools/repo_plugins/yum_dnf_src.py

```diff
@@ -17,6 +17,7 @@
 
 import dnf
 import hashlib
+import hawkey
```

Comment:
Is this a new dependency or do we use that already? I wasn't able to find this in the specfile.

Reply:
Nice catch. hawkey is provided by python3-hawkey which in turn is already a requirement of python3-dnf. Python3-dnf won't work without it so it will always be installed if python3-dnf is.

As we explicitly use it, I guess we should also put it in the spec file. What do you think?

Reply:
I just noticed that we also don't require python3-dnf or even dnf and just assume it is installed. Now I am not sure. Either we add all or none (dnf/python3-dnf/python3-hawkey).

Reply:
Thanks a lot for addressing this. I consider this resolved.